packages/airnode-validator/src/config/config.ts

```diff
@@ -86,8 +86,8 @@ export const chainOptionsSchema = z.discriminatedUnion('txType', [
     .object(
       {
         txType: z.literal('eip1559'),
-        baseFeeMultiplier: z.number().int().optional(),
-        priorityFee: priorityFeeSchema.optional(),
+        baseFeeMultiplier: z.number().int().optional(), // Defaults to BASE_FEE_MULTIPLIER defined in airnode-utilities
```

Comment:
I actually wanted to import the constants from airnode-utilities and use https://github.com/colinhacks/zod#default to make this explicit (because this can get out of sync easily if we make the field required).

I tried implementing that in https://github.com/api3dao/airnode/pull/1159/commits/46aed308ab237896b39cce6083c292a4616d4061 but I ended up reverting those changes, because lot of unit tests started to fail due to TS errors (because `default()` from zod makes the the field required and our fixtures needed to be updated).